extension/lib/magento/customer/getWishlistId.js

```diff
@@ -0,0 +1,20 @@
+const MagentoRequest = require('../Request')
+
+/**
+ * @param {Object} context
```

Comment:
You should also add params like "magentoUrl" to the config documentation
```suggestion
 * @param {StepContext} context
```